INSTALL.md

```diff
@@ -53,16 +53,18 @@ You will need:
  * GLEW
  * libpng
  * gettext
+ * librsvg2
  * libsndfile
  * libvorbis
  * libogg
  * OpenAL (OpenAL-Soft)
  * PhysFS
  * oggenc (to generate music files)
+ * xmlstarlet
 
 On Ubuntu (and probably any other Debian-based system), you can use the following command to install all required packages:
 ```
-  $ apt-get install build-essential cmake libsdl2-dev libsdl2-image-dev libsdl2-ttf-dev libsndfile1-dev libvorbis-dev libogg-dev libpng-dev libglew-dev libopenal-dev libboost-dev libboost-system-dev libboost-filesystem-dev libboost-regex-dev libphysfs-dev gettext git po4a vorbis-tools
+  $ sudo apt-get update && sudo apt-get install --no-install-recommends build-essential cmake libsdl2-dev libsdl2-image-dev libsdl2-ttf-dev libsndfile1-dev libvorbis-dev libogg-dev libpng-dev libglew-dev libopenal-dev libboost-dev libboost-system-dev libboost-filesystem-dev libboost-regex-dev libphysfs-dev gettext git po4a vorbis-tools librsvg2-bin xmlstarlet
```

Comment:
The instruction assumes basic familiarity of the user with their own Linux distribution. Not everybody needs to use `sudo` (f.e. they are on root account) and they may not need `apt-get update`. `--no-install-recommends` is also not necessary, it's in CI to reduce the build time because apt-get might install additional packages which are "recommended" but completely not needed for Colobot.

As i've said earlier, `librsvg2-bin` and `xmlstarlet` are optional dependencies so they should not be listed in the command.

Reply:
--no-install-recommends needs for save for save user time and traffic. apt-get update need's only for pure installed OS.